modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,79 @@
+:_content-type: PROCEDURE
+
+[id='op-running-as-root-user-namespace_{context}']
+= Running as root in a user namespace
+
+With {pipelines-shortname}, to run pods in the user namespace, you must pass the annotations that CRIO expects. To add these annotations for all users, you use the `oc edit clustertask buildah` command to edit the `buildah` cluster task. Or, to add the annotations to a specific namespace, you export the cluster task as a task to that namespace.
+
+Use {pipelines-shortname} to run pods in the user namespace.
+
+.Prerequisites
+
+* Pass the annotations that CRIO expects.
+
+.Procedure
+
+. Use the `oc edit clustertask buildah` command to edit the buildah cluster task.
+
+. Add the annotations for all users.
+
+. Export the cluster task as a task to the namepsace.
+
+. Use the `tkn create` feature to automate the creation of a cluster task.
++
+[source,terminal,subs="attributes+"]
+----
+$ oc new-project test
+----
++
+[source,terminal,subs="attributes+"]
+----
+$ tkn task create --from=buildah
+----
++
+[source,terminal,subs="attributes+"]
+----
+$ oc edit task buildah
+----
+
+. Add annotations to the task.
++
+[source,yaml,subs="attributes+"]
+----
+annotations:
+    io.kubernetes.cri-o.userns-mode: "auto"
+    io.openshift.builder: "true"
+----
+
+. Get the nodename where the buildah pod is.
++
+[source,terminal,subs="attributes+"]
+----
+oc get pod -o wide
+----
+
+. Verify that {pipelines-shortname} run your buildah container as a root user.
++
+[source,terminal,subs="attributes+"]
+----
+$ oc debug nodes/nodename
+----
++
+[source,terminal,subs="attributes+"]
+----
+$ chroot /host
+----
++
+[source,terminal,subs="attributes+"]
+----
+$ lsns -t user
+----
```

Comment:
```suggestion

. List the user namespaces. <Harsh, please tell the user what to look for in the list of user namespaces.>
+
[source,terminal,subs="attributes+"]
----
$ lsns -t user
----

```

Reply:
Please replace `<Harsh, please tell the user what to look for in the list of user namespaces.>` with an explanation.